02.calendar/cal.js

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env node
+
+import { fileURLToPath } from "url";
+import process from "process";
+import parseArgs from "minimist";
+
+export function execCalendar(options) {
```

Comment:
```suggestion
export function execCalendar(options, today = new Date()) {
```

というようにシステム日付を引数で渡せるようにするとsinonのようなモックライブラリを使わずに済みます

Reply:
テスト時はnew Date()のような動的な値ではなく静的な値で確認した方がよいと以前のプラクティスでご指摘頂いていましたので、sinonを使ってなんとか実現していましたが、引数で日付を渡すとシンプルでいいですね🙏
315a364 にて対応しました！